### ISSUE TITLE
Accuracy of Edge detection 

### ISSUE THREAD

**Issue by Li-yachuan** [GuHuangAI/DDM-Public#12]
Hi,
The structure of this model is different from DiffusionEdge, but the accuracy is totally same. How it works?
Adaptive FFT-filter is not mentioned in this work, is it used?

**Comment by GuHuangAI** [comment]
Yes, we use the adaptive FFT and report the result of DiffusionEdge.
In fact, the model in this repo uses a different noise scheduler and can obtain better results. Please see the codes for details.

**Comment by Li-yachuan** [comment]
Got it. thanks for your reply.



---Original---
From: ***@***.***&gt;
Date: Thu, Dec 5, 2024 14:25 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [GuHuangAI/ADM-Public] Accuracy of Edge detection  (Issue #12)




 
Yes, we use the adaptive FFT and report the result of DiffusionEdge.
 In fact, the model in this repo uses a different noise scheduler and can obtain better results. Please see the codes for details.
